enterprise/server/remote_execution/platform/platform.go

```diff
@@ -81,6 +82,7 @@ const (
 	disablePredictedTaskSizePropertyName = "debug-disable-predicted-task-size"
 	extraArgsPropertyName                = "extra-args"
 	EnvOverridesPropertyName             = "env-overrides"
+	EnvOverridesBase64PropertyName       = "env-overrides-base64"
```

Comment:
in the docs for this, when you write them, worth documenting which takes precedence if both set the same value

Reply:
Will add a doc for this in a follow-up PR. I am thinking between https://www.buildbuddy.io/docs/secrets or https://www.buildbuddy.io/docs/rbe-microvms#using-custom-images but not sure yet which one is best.